magento/magento2
# Added token revocation for REST API

### Description (*)

Created new  token revocation endpoint for REST API: ` /V1/integration/customer/revoke-customer-token`. 
Feature have similar functionality like here https://devdocs.magento.com/guides/v2.3/graphql/mutations/revoke-customer-token.html. 

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#30820

### Manual testing scenarios (*)

1. Generate customer authorization token.
2. Send request to new created `/V1/integration/customer/revoke-customer-token` endpoint
3. Check if token was removed

### Questions or comments


### Contribution checklist (*)
